items:
- uid: com.microsoft.azure.management.datalake.store.uploader._intentional_exception
  id: _intentional_exception
  parent: com.microsoft.azure.management.datalake.store.uploader
  href: com.microsoft.azure.management.datalake.store.uploader._intentional_exception.yml
  langs:
  - java
  name: IntentionalException
  fullName: com.microsoft.azure.management.datalake.store.uploader.IntentionalException
  type: Class
  source:
    remote:
      path: azure-mgmt-datalake-store-uploader/src/test/java/com/microsoft/azure/management/datalake/store/uploader/IntentionalException.java
      branch: master
      repo: https://github.com/Azure/azure-sdk-for-java
    path: azure-mgmt-datalake-store-uploader/src/test/java/com/microsoft/azure/management/datalake/store/uploader/IntentionalException.java
    startLine: 12
  package: com.microsoft.azure.management.datalake.store.uploader
  summary: <p>An exception that we want our mocks to throw sometimes to test out various code paths. </p>
  syntax:
    content: public class IntentionalException
  inheritance:
  - java.lang.Object
  - CloudException
references: []
